common/models/create_task_request.go

```diff
@@ -39,11 +40,12 @@ func (c CreateTaskRequest) Definition() swagger.Definition {
 	return swagger.Definition{
 		Type: "object",
 		Properties: map[string]swagger.Property{
-			"task_name":      swagger.NewStringProperty(),
-			"environment_id": swagger.NewStringProperty(),
-			"deploy_id":      swagger.NewStringProperty(),
 			// TODO: Change to Map defintion
```

Comment:
Would it make sense to address this Todo be addressed?

Reply:
I'm not sure about this comment; I haven't found `Definitions` that deviate fro this pattern elsewhere in our code, and as far as I can tell from the Swagger docs, there isn't a `"map"` `Type`.  I think it's just string, number, integer, boolean, array and object ([source](https://swagger.io/docs/specification/data-models/data-types/)). My best guess right now is that this TODO has already been accomplished, since `Properties` is already a `map[string]swagger.Property` and was forgotten to be removed.